test/unit/storage/entities/account.js

```diff
@@ -560,15 +560,15 @@ describe('Account', () => {
 			});
 
 			it('should fetch "productivity" with correct query', async () => {
-				const producedBlocks = 5;
+				const producedBlocks = 50;
 				const missedBlocks = 3;
 				const validAccount = new accountFixtures.Account({
 					producedBlocks,
 					missedBlocks,
 				});
 				await AccountEntity.create(validAccount);
-				const productivity = parseInt(
-					producedBlocks / (producedBlocks + missedBlocks) * 100.0
+				const productivity = parseFloat(
+					(producedBlocks / (producedBlocks + missedBlocks) * 100.0).toFixed(2)
```

Comment:
Personally I prefer to hardcode expected values.

Reply:
What's the point of `100.0` vs `100`?

Reply:
Good point. I will check how it was being returned before 1.5

Reply:
Just checked and it used to return `100` without decimals places. I will add more tests for that case.